Documentation/ApiOverview/DataFormats/T3datastructure/SheetReferences/Index.rst

```diff
@@ -12,6 +12,7 @@ choose to store one or more sheets externally in separate files. This
 is done by setting the value of the <[sheet ident]> tag to a relative
 file reference instead of being a definition of the <ROOT> element.
 
+..  todo: Does this still work? If so move example to examples extension
```

Comment:
Yes it works, I am using it in an extension (compatible with v11 and v12): https://github.com/brotkrueml/codehighlight/blob/main/Configuration/FlexForms/Options.xml